cyw43-firmware/Cargo.toml

```diff
@@ -0,0 +1,8 @@
+[package]
```

Comment:
do you like this approach? the alternative is to put back that gross C -> Rust code to extract the fake `.ihex` from a `.bin` firmware blob. I got rid of when I didn't trust it wasn't off by one or something in favor of this instead, up to you.